setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(name='swervedrive',
+      version='0.1',
+      description='FRC Team 4774\'s Swerve Drive library',
+      url='http://github.com/thedropbears/libswervedrive',
+      author='FRC Team 4774',
+      author_email='enquiries@thedropbears.org.au',
+      license='MIT',
+      packages=['swervedrive'],
+      install_requires=[
+          'numpy',
+      ],
+      zip_safe=False)
```

Comment:
I can't see any evidence that this is actually correct. Is it?

Reply:
Should it be true, or omitted completely?

Reply:
If you omit this, setuptools will try to determine the correct value automatically.

Reply:
This is also a legacy thing anyway, dating back to the days of egg distributions...